src/main/java/teammates/common/datatransfer/attributes/EntityAttributes.java

```diff
@@ -59,9 +59,12 @@ public boolean isValid() {
     public abstract void sanitizeForSaving();
 
     /**
-     * Adds non-empty error message to a List, ignores empty error messages.
+     * Adds {@code error} to {@code errors} if {@code error} is a non-empty string.
+     * @param error A String, which describes an error, can be empty if no error received.
```

Comment:
Thanks, I inserted this line and after that noticed that first tag @param after this line changed its color in eclipse - it has a little lighter blue than second @param tag. Is it normal behaviour?